### PR TITLE
OPHYKIKEH-7: improved end-user experience with radio buttons

### PR DESCRIPTION
**Pull request by mikhuttu**
Removed hover css from disabled radio button elements. Also changed cursor to `not-allowed` setting when inspecting a disabled radio button element. 

RadioButton elements are only used in `/yki/tutkintotilaisuudet` and `/ilmoittautuminen/tutkintotilaisuus/{id}` pages. Changes of this PR take place only in the former one given there can be no disabled elements in the latter.

### CODE REVIEW

File: src/main/js/src/components/UI/RadioButton/RadioButton.module.css [hunk 1]
```diff
@@ -14,19 +25,15 @@
 }
 
 .RadioButtonDisabled input {
-  margin-bottom: 0.5rem;
-  margin-right: 8px;
```
Comment (mikhuttu) [inline src/main/js/src/components/UI/RadioButton/RadioButton.module.css:18]: These were also defined under `.RadioButton input`.

File: src/main/js/src/setupProxy.js [hunk 2]
```diff
@@ -337,13 +337,17 @@ module.exports = function (app) {
...
+      try {
+        organizers.push(req.body);
+        res.send({ success: true });
+      } catch (err) {
+        printError(req, err);
+        res.status(404).send(err.message);
+      }
+    };
+
+    useLocalProxy ? proxyPostCall(req, res) : mockCall();
```
Comment (mikhuttu) [inline src/main/js/src/setupProxy.js:350]: Changed this logic to actually store organizer data if proxy backend is used. Nothing to do with the actual task.

File: src/main/js/src/components/UI/RadioButton/RadioButton.module.css (hunk 1 above)
Comment (mikhuttu) [inline src/main/js/src/components/UI/RadioButton/RadioButton.module.css:18]: These were also defined under `.RadioButton input`.

File: src/main/js/src/setupProxy.js (hunk 2 above)
Comment (mikhuttu) [inline src/main/js/src/setupProxy.js:350]: Changed this logic to actually store organizer data if proxy backend is used. Nothing to do with the actual task.

File: src/main/js/src/components/UI/RadioButton/RadioButton.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const radioButton = props => (
   <div
     className={[
       classes.RadioButton,
-      props.disabled ? classes.RadioButtonDisabled : '',
+      !props.disabled ? classes.RadioButtonEnabled : classes.RadioButtonDisabled,
```
Comment (P34m) [inline src/main/js/src/components/UI/RadioButton/RadioButton.js:10]: To make it clearer
- the exclamation mark can be removed
- if-else conditions can be reversed

File: src/main/js/src/components/UI/RadioButton/RadioButton.module.css [hunk 4]
```diff
@@ -14,19 +25,15 @@
 }
 
 .RadioButtonDisabled input {
```
Comment (P34m) [inline src/main/js/src/components/UI/RadioButton/RadioButton.module.css:27]: The following classes have the same CSS properties, so they can be combined with a comma.

.RadioButtonDisabled input
.RadioButtonDisabled .RadioButtonLabel
.RadioButtonDisabled .RadioButtonExtraLabel

**Review by P34m — CHANGES_REQUESTED** [review]
Looks good but needs some small changes.

**Pull request merged**